### PR TITLE
nginx 1.9.0

### PR DESCRIPTION
**Pull request by v-fearam**
https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.9.0/deploy/static/provider/cloud/deploy.yaml

### CODE REVIEW

File: cluster-manifests/ingress-nginx/deployment.yaml [hunk 1]
```diff
@@ -568,115 +571,101 @@ subjects:
...
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-        resources:
-          requests:
-            cpu: 100m
-            memory: 90Mi
-          limits:
-            cpu: 300m
-            memory: 100Mi
```
Comment (ckittel) [inline cluster-manifests/ingress-nginx/deployment.yaml:612]: Why did we get rid of these?

Comment (v-fearam) [reply cluster-manifests/ingress-nginx/deployment.yaml:612]: I was following the example and trying to make it work, let me try to added and test it again.

Comment (v-fearam) [reply cluster-manifests/ingress-nginx/deployment.yaml]: done

File: cluster-manifests/ingress-nginx/deployment.yaml [hunk 2]
```diff
@@ -568,115 +571,101 @@ subjects:
...
```
[the remark this answers is not in the document]
Comment (v-fearam) [reply cluster-manifests/ingress-nginx/deployment.yaml]: done

**Pull request merged**